snapshots/urbanization/2024-12-02/ghsl_urban_centers.xlsx.dvc

```diff
@@ -0,0 +1,33 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Global Human Settlement Layer Dataset -  Stats in the City Database
+    description: |-
+      The "Stats in the City Database" offers harmonized data on population and population density for 11,422 urban centres.
+
+      This data, based on the Global Human Settlement Layer Dataset, uses the Degree of Urbanisation framework to delineate spatial entities and integrates geospatial data from a variety of open-source datasets. It represents one of the most comprehensive resources for understanding urban population patterns and densities worldwide
+    date_published: "2024"
+
+    # Citation
+    producer: European Commission, Joint Research Centre (JRC)
+    citation_full: |-
+      Center For International Earth Science Information Network-CIESIN-Columbia University. 2018. “Gridded Population of the World, Version 4 (GPWv4): Population Count, Revision 11.” Palisades, NY: NASA Socioeconomic Data and Applications Center (SEDAC). https://doi.org/10.7927/H4JW8BX5
```

Comment:
In the report (https://human-settlement.emergency.copernicus.eu/documents/GHSL_UCDB_R2024.pdf), they use "only":

"European Commission, Joint Research Centre, Melchiorri, M., Mari Rivero, I., Florio, P., Schiavina, M., Krasnodebska, K.,
Politis, P., Uhl, J., Pesaresi, M., Maffenini, L., Sulis, P., Crippa, M., Guizzardi, D., Pisoni, E., Belis, C., Oom, D., Branco, A., Mwaniki, D., Githira, D.,
Kochulem, E., Tommasi, P., Carioli, A., Ehrlich, D., Kemper, T. and Dijkstra, L., Stats in the City the GHSL Urban Centre Database 2025,
Publications Office of the European Union, Luxembourg, 2024, https://data.europa.eu/doi/10.2760/3046391, JRC139768"

But your citation seems more comprehensive, so happy to go with that!

Reply:
that's what the source recommended via email so I'll stick with this one!